tests/test_signature.py

```diff
@@ -114,6 +114,35 @@ def test_expand_varargs():
     assert s.expand_varargs(4) == (int, int, float, float)
 
 
+def test_varargs_tie_breaking():
```

Comment:
Let's just add a reminder in here 

```suggestion
def test_varargs_tie_breaking():
    # Those are related to Bug #117
```